source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModel/AdaptiveCardsSharedModel.vcxproj

```diff
@@ -90,6 +90,7 @@
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
       <TreatWarningAsError>true</TreatWarningAsError>
       <DisableSpecificWarnings>4702</DisableSpecificWarnings>
+      <LanguageStandard>stdcpp17</LanguageStandard>
     </ClCompile>
```

Comment:
If this is necessary, shouldn't it also be included in the uwp version of the vcxproj?

Reply:
It's not strictly required, but since we're going to need to build C++17 on our end, we need to make sure this build remains happy. That being said, we should definitely switch UWP over. I'll push a change shortly.